README.md

```diff
@@ -1,2 +1,16 @@
-# log4a
+# Logger
+
 Logger for Asamm libraries and tools.
+
+## Quick start
+
+Add dependency to your `build.gradle` module config
+
+```gradle
+dependencies {
+     // get Logger
+     compile 'com.asamm:logger-asamm:[latest]'
```

Comment:
[suggestion] `logger-asamm` -> `asamm-logger`

Reply:
😃 , oki, I'll name this dependency just a logger.